test/OrchardCore.Commerce.Tests.UI/Tests/ProductTests/ProductBehaviourTests.cs

```diff
@@ -42,4 +50,7 @@ public ProductBehaviourTests(ITestOutputHelper testOutputHelper)
             context.Driver.Exists(By.XPath($"//li[contains(text(), 'PriceVariantsProduct: Small')]").Visible());
         },
         browser);
+
+    private static void ShoppingCartItemCountShouldBe(UITestContext context, string count) =>
+        context.Get(By.ClassName("item-count")).Text.ShouldBe(count);
```

Comment:
Avoid using number strings. You can make the parameter `int` and use `.ShouldBeAsString(count)`.